include/seastar/rpc/rpc_types.hh

```diff
@@ -432,7 +432,29 @@ struct tuple_element<I, seastar::rpc::tuple<T...>> : tuple_element<I, tuple<T...
 template <> struct fmt::formatter<seastar::rpc::connection_id> : fmt::ostream_formatter {};
 #endif
 
-#if FMT_VERSION >= 100000
+#if FMT_VERSION < 100000
```

Comment:
I didn't understand why this if is needed. If everything worked on FMT<100000 before this patch, why do we suddenly need to this?

Reply:
> everything worked on FMT<100000 before this patch 

not quite true. not everything worked out of the box. this does not work without `FMT_DEPRECATED_OSTREAM` defined.

Reply:
i amended the commit message to add this information to it.

Reply:
Thanks. Clearly this fix is for a very different problem from what the original commit message claimed it was fixing.